modules/apps/object/object-js-components-web/types/src/main/resources/META-INF/resources/utils/api.d.ts

```diff
@@ -62,6 +62,7 @@ export interface NotificationTemplate {
 declare type ObjectRelationshipType = 'manyToMany' | 'oneToMany' | 'oneToOne';
 interface ObjectRelationship {
 	deletionType: string;
+	treeEdge: boolean;
```

Comment:
hey @danvb112 could you please sort this parameter?

Thanks!

Reply:
Thanks for having notice that Feliphe! Done!